cache-filter/src/utils.rs

```diff
@@ -33,31 +39,32 @@ pub fn do_auth_call<C: proxy_wasm::traits::HttpContext>(
     let service = Service::new(request_data.service_id.as_ref(), cred);
 
     // TODO: Change this consider every app identifiers
-    let app = threescalers::application::Application::from_user_key(
-        request_data.app_id.as_string().as_ref(),
-    );
+    let app;
+    match &request_data.app_id {
+        AppIdentifier::UserKey(user_key) => app = Application::from_user_key(user_key.as_ref()),
+        AppIdentifier::AppId(app_id, None) => app = Application::from_app_id(app_id.as_ref()),
+        AppIdentifier::AppId(app_id, Some(app_key)) => {
+            app = Application::from_app_id_and_key(app_id.as_ref(), app_key.as_ref())
+        }
+    }
```

Comment:
Looks great! Just remove the comment as it's not needed anymore. :)

Reply:
fixed